azure-functions/src/generic.rs

```diff
@@ -41,6 +41,10 @@ impl From<TypedData> for Value {
     fn from(data: TypedData) -> Self {
         match data.data {
             None => Value::None,
+            Some(Data::CollectionBytes(_c)) => unimplemented!(),
```

Comment:
How should this be represented? Does value need additional types?

Reply:
I actually have this implemented in a WIP branch where I'm implementing Durable Functions 2.0.

Is updating the protobuf necessary for this PR or can it wait for that feature work?

Reply:
Can wait for that feature work.